portfolio/src/main/webapp/script.js

```diff
@@ -61,3 +61,171 @@ function addRandomFunFact() {
 
 }
 
+/* 
+ * Creates the pagination bar given the number of pages and the previous active page number.
+ * Returns the current active page number.
+ */
+function createPagination(numPages, activePageNumber) {
```

Comment:
Please add JSDocs for the params and return values, here and everywhere below.

Eg:https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Reply:
Got it! Should have saw this earlier.